xla/stream_executor/cuda/cuda_compute_capability.h

```diff
@@ -96,6 +96,8 @@ struct CudaComputeCapability {
     return major >= CudaComputeCapabilities::BLACKWELL;
   }
 
+  bool IsBlackwell() const { return *this == Blackwell(); }
```

Comment:
I can't say I'm a huge fan of the name `IsBlackwell` given devices with compute capability 12.0 also are Blackwell, and yet this returns false for them. But I cannot think of a better name, so I'll accept this for now.